src/shared/lib/oql/oqlfilter.ts

```diff
@@ -913,8 +913,8 @@ export function filterCBioPortalWebServiceDataByOQLLine(
 export function filterCBioPortalWebServiceDataByOQLLine(
     oql_query: string,
     data:
-        | ((AnnotatedMutation | NumericGeneMolecularData)[])
```

Comment:
[Don't require parens for same-operator logical expressions](https://prettier.io/blog/2019/11/09/1.19.0.html#dont-require-parens-for-same-operator-logical-expressions-6864-by-jridgewell)